### DISCUSSION TITLE
Return 204 when using aiohttp request

### DISCUSSION THREAD

**Discussion by solo-xwz** [aio-libs/aiohttp#6425] [Q&A]
I use `aiohttp` to make crawler requests. For the same request using `httpx` or `requests` returns normal content, but aiohttp returns empty, why?

**Comment by asvetlov** [comment]
Is 204 described as the status code?

[204 No Content](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204) should return no content by definition.

**Reply by solo-xwz** [reply]
> Is 204 described as the status code?
> 
> [204 No Content](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204) should return no content by definition.

yes,status code is 204
